tests/Resources/ti.ui.view.addontest.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2020 by Axway, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+const should = require('./utilities/assertions');
+
+describe('Titanium.UI.View', () => {
+
+	it.android('borderRadiusEdges', () => {
+		const view = Ti.UI.createView({
+			width: 100,
+			height: 100,
+			borderRadiusCorners: [
```

Comment:
Shouldn't this give an exception as this property is valid with 'Titanium.UI.View.borderRadius'.

Reply:
I don't think it should, we don't when setting `borderWidth`; this is just a test to see if the properties get set.